### PR TITLE
Added DuplicateKey string to check name message.

### LINKED ISSUE #120: DuplicatedKey: Show check name in message
Right now the message looks like this:
```
.env:2 The FOO key is duplicated
```

Need to add the check name to the message:
```
.env:2 DuplicatedKey: The FOO key is duplicated
```

### PR DESCRIPTION
**Pull request by SaMuRa1ReM1X**
Fixes #120 

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [ ] Docs have been added / updated (for bug fixes / features).

### CODE REVIEW

File: src/checks/duplicated_key.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@ impl Default for DuplicatedKeyChecker {
     fn default() -> Self {
         Self {
             keys: HashSet::new(),
-            template: String::from("The {} key is duplicated"),
+            template: String::from("DuplicatedKey: The {} key is duplicated"),
```
Comment (mgrachev) [inline src/checks/duplicated_key.rs:14]: In the future I would like to have the ability to disable some checks by their names.
To do that, all checks should have the `name` field.
Please, move the check's name from the template to the special field.
Also, I would like to have a method to show the check's name in the message, for example:
```rust
impl DuplicatedKeyChecker {
    fn message(&self, key: &str) -> String {
      // code...
    }
}
```

**Comment by mgrachev** [comment]
@SaMuRa1ReM1X Thanks a lot for your contribution! ❤️

I have left a comment. Could you please fix it?

File: src/checks/duplicated_key.rs (hunk 1 above)
Comment (SaMuRa1ReM1X) [reply src/checks/duplicated_key.rs:14]: I'll implement the message function as specified. Sorry for the delayed response.

**Comment by mgrachev** [comment]
@SaMuRa1ReM1X Thank you! 🚀

**Pull request merged**